changeItem.php

```diff
@@ -0,0 +1,29 @@
+<?php
+require_once 'config.php';
+
+function changeItem(array $newTodoItem): array
+{
+    if (file_exists(TODO_FILE) && array_key_exists('id', $newTodoItem)) {
+        $todolist = json_decode(file_get_contents(TODO_FILE), true);
+
+        foreach ($todolist as &$todo) {

```

Comment:
if you use links - unset them after each iteration to release the memory and avoid any unexpected behaviour